up-and-running/customization.md

```diff
@@ -1,31 +1,145 @@
+
+The secret key used both between internal services and to connect
+upstream replication.
+
+### Storage
+
+Configure the storage for npm Enterprise's various data-files. Frequently you
+will need to switch these paths to match a larger external drive that
+you've mounted on your appliance.
+
+### Scoped CLI Search
+
+By default `npm search` does not return scoped packages (since this is
+  potentially a breach of private information). Enable this setting to
+start returning scoped search results to the CLI.
+
+_Note: you should only do this if your npm Enterprise appliance is
+  running behind a firewall._
+
+### Read Through Cache
+
+By default, rather than mirroring the entire `>1TB` npm registry,
+npm Enterprise selectively caches packages the first time that you
+install them:
+
+```
+npm i lodash --registry=http://my-private-registry:8080
+```
+
+If you would rather selectively whitelist packages from the public registry,
+or would prefer to mirror the entire registry, set this option to `No`.
+
+### Reject Unauthorized
+
+This this option to `No`, if you would like to disable strict SSL checks
+for operations performed by npm Enterprise.
+
+_As an example, you might disable SSL checks if your upstream GitHub enterprise
+server is using a self-signed certificate._
+
+### Upstream Registry
+
+What upstream registry should npm Enterprise replicate packages from? This setting
+defaults to the public registry (`https://replicate.npmjs.com`), but can be
+configured to replicate from other internal npm Enterprise registries for HA
+deployments.
+
+* **Upstream URL**: the registry to replicate from.
+* **Upstream secret**: if connecting to another npm Enterprise registry, provide
+  the _Secret used between services_ of the upstream server.
+* **Policy to apply during replication**: defaults to `white-list`, only replicating
+  modules added to your whitelist file (either by an administrator, or by the
+  read-through-cache). To replicate an entire upstream registry, set this value
+  to `mirror`.
+
+### Auth Scoped Installs
+
+Should authentication be required to install scoped modules?
+
+### Auth All Installs
+
+Should authentication be enabled to install both scoped (`@company/foo`), and
+global modules from your registry (`lodash`).
+
+### Auth Website
+
+Should authentication be required to view the npm Enterprise website.
+
+### Publication Settings
+
+The default `All` setting, allows both scoped and global modules to be
+published to your npm Enterprise registry.
+
+By setting this value to `Scoped`, only scoped modules will be allowed to
+be published to your registry.
+
+For replicas in HA deployments, you can set this value to `Read Only` to
+disable all publications.
+
+### Authentication
+
+Rather than handling user accounts itself, npm Enterprise allows
+you to connect to your company's existing authentication system.
+
+Read more about [configuring authentication].
+
+[configuring authentication]: /up-and-running/auth/
```

Comment:
This link should probably be `../up-and-running/auth/README.md` or `auth/README.md`.